src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -103,6 +103,7 @@ CacheManager::CacheManager()
     : CacheManagerInterface()
     , pt_(new policy_table::Table)
     , backup_(new SQLPTRepresentation())
+    , settings_(nullptr)
```

Comment:
Something I noticed with this initialization. `settings_` is assumed to be initialized here: https://github.com/smartdevicelink/sdl_core/blob/6182ca86dae52ab93fb9c4a82a4c1b1d7ef89336/src/components/policy/policy_regular/src/cache_manager.cc#L830

`settings_` should be replaced with `get_settings()` in this case, since it includes a `DCHECK` to cover the case where `settings_` isn't initialized for some reason.